content/en/blog/2025/stability-proposal-announcement.md

```diff
@@ -0,0 +1,265 @@
+---
+title: Evolving OpenTelemetry’s Stabilization and Release Practices
+linkTitle: Stability Proposal Announcement # Mandatory, make sure that your short title.
+date: 2025-10-23 # Put the current date, we will keep the date updated until your PR is merged
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Austin Parker](https://github.com/austinlparker) (honeycomb.io)
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+body_class: otel-with-contributions-from # TODO: remove this line if there are no secondary contributing authors
+issue: the issue ID for this blog post # TODO: See https://opentelemetry.io/docs/contributing/blog/ for details
+sig: Governance Committee # TODO: add the name of the SIG that sponsors this blog post
+---
+
+<!-- If your post doesn't have secondary authors, then delete the following paragraph: -->
+
+With contributions from, and on behalf of, the OpenTelemetry Governance
+Committee.
+
+## Summary
+
+OpenTelemetry is changing how the project approaches stability and releases,
+across all sub-projects and repositories.
+
+**Why?**
+
+- User interviews, surveys, and discussions have demonstrated that the
+  complexity of OpenTelemetry creates impediments to production deployments.
+
+**What's Changing?**
+
+- We'll be moving to a simpler, but stricter, definition of what components can
+  be present in a stable release and updating stability requirements to include
+  new guidance on documentation, performance testing, etc.
+- Semantic convention guidelines will be updated to encourage faster iteration,
+  greater federation, and to unblock consumers to allow stable instrumentation
+  releases.
+- To make these releases easier to consume, we'll be introducing 'epoch
+  releases' -- similar to the Kubernetes release cycle.
+
+**What we need your input on:**
+
+- Implementation timelines and stability requirements.
+- Proposed changes to semantic convention stability.
+- How we can make releases better.
+
+[Join the discussion!](https://github.com/open-telemetry/community/discussions/3098).
+
+## Why are we doing this?
+
+OpenTelemetry is a large, complex, and highly involved project. A quick glance
+at the
+[specification compliance matrix](https://github.com/open-telemetry/opentelemetry-specification/blob/main/spec-compliance-matrix.md)
+should be enough to tell you that supporting tracing, metrics, logs, and
```

Comment:
This para should be re-written. "should be enough to tell you" reads a bit odd - I suggest changing to "A quick glance at the specification <link> shows you that supporting tracing, metrics, logs and profiles across different languages and environments is complex".